packages/frontend/ci/configFromEnvironment.js

```diff
@@ -0,0 +1,70 @@
+const assert = require("assert");
+
+const Environments = require("../../../features/environments.json");
+const { parseBooleanFromShell } = require("../src/config/envParsers");
+
+const Config = {
+  BRANCH: process.env.BRANCH,
+  CLOUDFLARE_BASE_URL: process.env.CLOUDFLARE_BASE_URL,
+  CONTEXT: process.env.CONTEXT,
+  DEBUG_BUILD: process.env.DEBUG_BUILD,
```

Comment:
Please use `parseBooleanFromShell()` on this and remove the coercion from `ParcelBundler.js`

Reply:
@esaminu I believe this is the coercion that Daryl is referring to: https://github.com/near/near-wallet/blob/master/packages/frontend/ci/ParcelBundler.js#L13 - with your parsing change we can now directly refer to `DEBUG_BUILD` 🎉

Reply:
Done! Thanks for catching 😄